_includes/scenarios/integrate/read-attribute-from-peer.md

```diff
@@ -0,0 +1,249 @@
+
+### Reject a ReadAttributeRequestItem
+
+Even if the Recipient accepts the [Request for reading Attributes]({% link _docs_integrate/read-attribute-from-peer.md %}#request-for-reading-attributes) as a whole, it may decide not to share every requested Attribute with the Sender. To be more precise, the Recipient has the option of rejecting [ReadAttributeRequestItems]({% link _docs_integrate/read-attribute-from-peer.md %}#description-of-readattributerequestitem) that have the value `false` specified in their `mustBeAccepted` property. To reject a ReadAttributeRequestItem, use the following [parameter]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem-response-parameters):
+
+| Property | Value   |
+| -------- | ------- |
+| accept   | `false` |
+
+The rejection of a ReadAttributeRequestItem leads to the creation of a corresponding [ResponseItem]({% link _docs_integrate/data-model-overview.md %}#responseitem) of type [RejectResponseItem]({% link _docs_integrate/data-model-overview.md %}#rejectresponseitem). This will be contained within the `items` property of the [Response]({% link _docs_integrate/data-model-overview.md %}#response) to the [Request for reading Attributes]({% link _docs_integrate/read-attribute-from-peer.md %}#request-for-reading-attributes).
+
+In addition to [AcceptResponseItems]({% link _docs_integrate/data-model-overview.md %}#acceptresponseitem) and [RejectResponseItems]({% link _docs_integrate/data-model-overview.md %}#rejectresponseitem), [ErrorResponseItems]({% link _docs_integrate/data-model-overview.md %}#errorresponseitem) are one of the three forms of [ResponseItems]({% link _docs_integrate/data-model-overview.md %}#responseitem). These are never created manually, but can occur due to an error.
+{: .notice--info}
+
+### Example of accepting a RequestItemGroup
+
+Let's look at an example where the Sender is interested in the Recipient's [BirthDate]({% link _docs_integrate/attribute-values.md %}#birthdate) and contact information in the form of an [EMailAddress]({% link _docs_integrate/attribute-values.md %}#emailaddress) or a [PhoneNumber]({% link _docs_integrate/attribute-values.md %}#phonenumber). To ask the Recipient for this data, the Sender creates a [Request for reading Attributes]({% link _docs_integrate/read-attribute-from-peer.md %}#request-for-reading-attributes), which contains a [ReadAttributeRequestItem]({% link _docs_integrate/read-attribute-from-peer.md %}#description-of-readattributerequestitem) belonging to the BirthDate and a [RequestItemGroup]({% link _docs_integrate/read-attribute-from-peer.md %}#read-multiple-attributes-with-a-requestitemgroup) belonging to the contact information in its `items` property. The RequestItemGroup itself contains two ReadAttributeRequestItems in its `items` property, namely one for the EMailAddress and one for the PhoneNumber.
```

Comment:
Like above, I'd not use the technical terms in the text here, but "birth date"...  (also in the following)

Reply:
In some places I have replaced the technical terms with the normal words (if the linked Attribute Values have not been used in a technical example).